embedded-ebpf.go

```diff
@@ -0,0 +1,12 @@
+//go:build ebpf
+// +build ebpf
+
+package tracee
+
+import (
+	"embed"
+)
+
+//go:embed "dist/tracee.bpf/*"
```

Comment:
This is needed if we want to have independent compilations of "tracee-ebpf" and "tracee-rules" (and we do). That is why I created the "ebpf" go tag.